app/src/pages/playground/schemas.ts

```diff
@@ -0,0 +1,76 @@
+import { z } from "zod";
+
+import {
+  ImageAttributesPostfixes,
+  LLMAttributePostfixes,
+  MessageAttributePostfixes,
+  MessageContentsAttributePostfixes,
+  SemanticAttributePrefixes,
+} from "@arizeai/openinference-semantic-conventions";
+
+import { CHAT_MESSAGE_ROLES } from "@phoenix/store";
```

Comment:
since we are parsing from the conventions and we have no idea about the type of strings, I think we might just have to treat the role as a string with a fallback. It's more honest that way

Reply:
that makes sense, since the playground now requires these to be of this type, I think if we parse as string we have two options:
1. try to perform some semi-reasonable mapping from string role to role enum
```typescript
function convertStringToRole(roleString) {
  if (role.includes("user") || role.includes("human")) {
  return CHAT_MESSAGE_ROLES.user
  // ...
  return DEFAULT_ROLE
  }
}
```
2. fallback to default for any role that doesn't exactly match

I guess if you already changed assistant to ai, 1 seems like a better approach